Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -707,6 +707,22 @@ extension NavigatorIndex {
                 multiCuratedUnvisited.remove(normalizedIdentifier)
             }
             
+            // Check if the render node has a variant for Objective-C
+            let objCVariantTrait = renderNode.variantOverrides?.values.flatMap({ $0.traits }).first { trait in
```

Comment:
This code assumes that the primary variant is Swift, FWIW. It'd be more flexible if it just checked for other variants besides the primary variant.

Also, it assumes there are only two variants, swift and objective-c, but I understand that's a larger problem to resolve.

Reply:
Couldn't this just loop over all the `variantOverrides` instead of reaching into it for a particular one?

Reply:
For now, this is just indexing the variant overrides for Objective-C. Supporting an arbitrary set of languages would be a larger task.